added toast message, on Submit functionality, Contact Form

HarrisMBrad opened the pull request:
setting up for the message as a toast

HarrisMBrad committed:
added toast message, on Submit functionality, Contact Form

HarrisMBrad committed:
component displays a toast notification with a message fetched from an API

missatrox44 commented:
Looking good!!

- remove two 'Shut Up!' messages on home page
<img width="368" alt="Screenshot 2023-08-01 at 3 21 26 PM" src="https://github.com/FatherWolf/ibginstitute-react/assets/74509058/f693bb22-6e07-411b-b35a-11a8d7715bba">

HarrisMBrad commented:
Recent Changes (Commit: Contact Form #45)
Added toast message

Implemented a Toast component that fetches messages from an API and displays them as a toast notification with a delay of 3 seconds. The API function used here is a placeholder; replace it with the actual function to fetch the data.

On Submit functionality

Added onSubmit functionality to the form that triggers the API call when the form is submitted. (Provide more details about what this feature does)
Contact Form #45

This update includes the addition of a new contact form (Form #45). (Describe what the form does, what inputs it accepts, and any other relevant details)
Past Changes
Commit Name: A short description of what changes were made in this commit.
Commit Name: A short description of what changes were made in this commit.
Components
SuccessToast Component
This component fetches a message from an API and displays it as a toast notification after a delay of 3 seconds. For more information, refer to the [dedicated section](#success-toast-component).

Detailed Descriptions
SuccessToast Component
This component is a React component that fetches a message from an API and displays it as a toast notification after a delay of 3 seconds.


We use the useState hook to create a state variable (myMessage) and a function to update it (setMyMessage). Inside the useEffect hook, we call the fetchResponse function when the component mounts. This function fetches the message and updates the state with setMyMessage. The toast notification is created by calling toast(message). However, we delay this by 3 seconds using the setTimeout() function. In the component's return statement, we include a ToastContainer. This is a requirement of the react-toastify library to hold the toast notifications. This component fetches a message when it mounts, waits for 3 seconds, and then displays it in a toast notification.

HarrisMBrad committed:
removed isSubmitted state on the contact form

HarrisMBrad committed:
re-arranged the load call to fire on click of the contact form, WIP
